### ISSUE TITLE
token_endpoint_auth_signing_alg_values_supported[11]: unknown variant `ES256K`, expected one of `HS256`, `HS384`, `HS512`, `RS256`, `RS384`, `RS512`, `ES256`, `ES384`, `ES512`, `PS256`, `PS384`, `PS512`, `none`

### ISSUE THREAD

**Issue by chettoy** [ramosbugs/openidconnect-rs#99]
```
OpenID Connect Discovery failed.

Caused by:
    0: Failed to parse server response
    1: token_endpoint_auth_signing_alg_values_supported[11]: unknown variant `ES256K`, expected one of `HS256`, `HS384`, `HS512`, `RS256`, `RS384`, `RS512`, `ES256`, `ES384`, `ES512`, `PS256`, `PS384`, `PS512`, `none` at line 1 column 775
    2: unknown variant `ES256K`, expected one of `HS256`, `HS384`, `HS512`, `RS256`, `RS384`, `RS512`, `ES256`, `ES384`, `ES512`, `PS256`, `PS384`, `PS512`, `none` at line 1 column 775
```
After reading the relevant issues I learned that I need to extend these definitions with types based on the OIDC providers. I tried to copy `CoreJwsSigningAlgorithm` to `CustomJwsSigningAlgorithm` and found that it seems that more files need to be customized. I don't think I can do it. What should I do?

**Comment by ramosbugs** [comment]
Hey @chettoy, I pushed a fix for the 2.x release branch in https://github.com/ramosbugs/openidconnect-rs/commit/0db8ad1e7b5a156cd4a649e760135154317910f5. I haven't cut a release yet, but can you try this out and see if it fixes your issue?

Try this in `Cargo.toml`:
```toml
openidconnect = { git = "https://github.com/ramosbugs/openidconnect", rev = "0db8ad1e7b5a156cd4a649e760135154317910f5" }
```

**Comment by chettoy** [comment]
It works!

**Comment by ramosbugs** [comment]
This is now released in [2.5.0](https://crates.io/crates/openidconnect/2.5.0)!
